src/internal/channel.js

```diff
@@ -13,9 +13,21 @@ export function emitter() {
     return () => remove(subscribers, sub)
   }
 
+  function emitAll(items) {
+    for (var i = 0, len = items.length; i < len; i++) {
+      emit(items[i])
+    }
+  }
+
   function emit(item) {
+    if (item === undefined) {
```

Comment:
its only guarding from the `undefined`, but I think `null` should filtered out too, cant use simple `!item` though as we need to pass i.e. `0` (in case somebody emit numbers)